lib/matplotlib/axes/_axes.py

```diff
@@ -5112,6 +5174,15 @@ def reduce_C_function(C: array) -> float
 
             if zscale == "log":
                 bin_edges = np.geomspace(zmin, zmax, nbins + 1)
+            elif zscale == "symlog":
+                zmin, zmax = symlog_transform[zname].transform_non_affine(
+                    np.array([zmin, zmax])
+                )
+                bin_edges = np.linspace(zmin, zmax, nbins + 1)
+                bin_edges = inv_symlog_trans[zname].transform_non_affine(
+                    bin_edges
+                )
```

Comment:
Fits in one line:
```suggestion
                bin_edges = inv_symlog_trans[zname].transform_non_affine(bin_edges)
```

Reply:
Not if you care about line length < 80

Reply:
Matplotlib cares about line length < 89
https://matplotlib.org/devdocs/devel/coding_guide.html#pull-request-guidelines